### PR TITLE
[Merged by Bors] - chore: remove duplicate imports from five files

### PR DESCRIPTION
**Pull request by euprunin**

### CODE REVIEW

File: Mathlib/RingTheory/Flat/Basic.lean [hunk 1]
```diff
@@ -3,17 +3,16 @@ Copyright (c) 2020 Johan Commelin. All rights reserved.
...
-import Mathlib.Algebra.Module.Projective
-import Mathlib.Algebra.Module.Injective
+import Mathlib.Algebra.DirectSum.Module
+import Mathlib.Algebra.Exact
 import Mathlib.Algebra.Module.CharacterModule
+import Mathlib.Algebra.Module.Injective
+import Mathlib.Algebra.Module.Projective
 import Mathlib.LinearAlgebra.DirectSum.TensorProduct
 import Mathlib.LinearAlgebra.FreeModule.Basic
-import Mathlib.Algebra.Module.Projective
```
Comment (grunweg) [inline Mathlib/RingTheory/Flat/Basic.lean:14]: This import was (syntactically) present twice; the other changes are just sorting imports alphabetically.

**Review by grunweg — COMMENTED** [review]
Thanks for finding and correcting this!
To clarify (for other reviewers): these imports are not transitive, but syntactically duplicate - so removing these is obviously useful. And sorting imports alphabetically also doesn't hurt.
maintainer merge

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**